frontend/src/components/MentorList.js

```
@@ -1,362 +1,163 @@
-    // 검색 필터링
     if (searchTerm) {
-      result = result.filter(mentor => 
-        mentor.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
-        mentor.role.toLowerCase().includes(searchTerm.toLowerCase()) ||
-        mentor.company.toLowerCase().includes(searchTerm.toLowerCase()) ||
-        mentor.tags.some(tag => tag.toLowerCase().includes(searchTerm.toLowerCase()))
+      const term = searchTerm.toLowerCase();
+      result = result.filter(m =>
+          (m.title && m.title.toLowerCase().includes(term)) ||
+          (m.introduction && m.introduction.toLowerCase().includes(term)) ||
+          (m.category && m.category.toLowerCase().includes(term))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)